### ISSUE TITLE
Documentations

### ISSUE THREAD

**Issue by Nic062** [mmarquezs/My.Jdownloader-API-Python-Library#18]
Hello,

Where i can find the documentations ? 
I try to make it but nothing.

Yours sincerely.

**Comment by mmarquezs** [comment]
Yeah it seems it's broken. Anyway the documentation is generated from the code, basically from the comments in each function and so, so  you can check the code. But don't expect a great documentation and there are probably mistakes.

**Comment by Mizio66** [comment]
Hi, is this still working with JD? Trying to find out if it is this or JD. no links added.

gracias

**Comment by mmarquezs** [comment]
It does, there are features missing but adding links should work.
If you have problems feel free to open a new issue and I can help you there if you give examples of what you have tried.
This issue was so I would fix the readthedocs stuff at some point. But I haven´t looked into it because the docs are already in the code and they aren´t very good anyway. Plus most of the information required to do requests is of the official API itself not the library.

**Comment by luzpaz** [comment]
Docs are not available

**Comment by mmarquezs** [comment]
Yeah as it says it the README, the docs are not working currently, I could take a look at it at some point, but tbh the docs that existed were (and would be again if fixed) simply the docstrings in the code, so you can check those in the code itself, it would be the same docs, just slightly less easy to read.

Apart from that, there is also the MyJDownloader docs, https://my.jdownloader.org/developers/ , which the naming for the methods and so on should be similar, (they aren't great either tbh, sometimes is not clear what exactly you have to sent).
